plugins/scaffolder-actions/scaffolder-backend-module-utils/README.md

```diff
@@ -195,3 +196,41 @@ spec:
         path: ${{ parameters.path }}
         content: ${{ parameters.content }}
 ```
+
+### Example of using parseFile action
+
+```yaml
+---
+apiVersion: scaffolder.backstage.io/v1beta3
+kind: Template
+metadata:
+  name: parse-file-template
+  title: Parse From File
+  description: Example temaplte to parse from a file with on the given path with the given content in the workspace.
+spec:
+  owner: roadie
+  type: service
+
+  parameters:
+    - title: Parse From File
+      properties:
+        path:
+          title: Path
+          type: string
+          description: The path to the file
+        parser:
+          title: Parser
+          type: string
+          enum:
+            - yaml
+            - json
+            - multiyaml
+          description: The content to parse from the file
+  steps:
+    - id: parsefile
+      name: Parse File
+      action: roadiehq:utils:fs:parse
+      input:
+        path: ${{ parameters.path }}
+        parser: ${{ parameters.parser }}
+```
```

Comment:
Can you add a section for all of the actions that you are adding with this change?